### PR TITLE
opensnitch-ui: Add wayland support

### PR DESCRIPTION
**Pull request by sochotnicky**
Without this dependency opensnitch needs to be run with QT_QPA_PLATFORM=xcb override in Wayland environments.

###### Description of changes

This just adds a dependency to allow running opensnitch-ui natively under Wayland.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/networking/opensnitch/ui.nix [hunk 1]
```diff
@@ -33,6 +34,7 @@ python3Packages.buildPythonApplication rec {
     pyinotify
     notify2
     pyasn
+    qtwayland
```
Comment (Artturin) [inline pkgs/tools/networking/opensnitch/ui.nix:37]: Please add it to `buildInputs`, `wrapQtAppsHook` will pick it up from there so it's unnecessary to propagatedBuildInputs it.

Comment (sochotnicky) [reply pkgs/tools/networking/opensnitch/ui.nix:37]: Done

Comment (Artturin) [reply pkgs/tools/networking/opensnitch/ui.nix:37]: You forgot to push

**Comment by onny** [comment]
Just tested it and it works fine, thank you! Waiting for your fix

File: pkgs/tools/networking/opensnitch/ui.nix (hunk 1 above)
Comment (sochotnicky) [reply pkgs/tools/networking/opensnitch/ui.nix]: Oof, sorry.

Comment (sochotnicky) [reply pkgs/tools/networking/opensnitch/ui.nix]: It looks like you fixed it up in the meantime so I'll leave this be?

**Pull request merged**